src/rest/Query.ts

```diff
@@ -5,6 +5,16 @@ import { Context } from '../ui/PipelineContext/PipelineGlobalExports';
 import { ICategoryFacetRequest } from './CategoryFacetRequest';
 import { IFacetRequest } from './Facet/FacetRequest';
 
+/**
+ * The available global configuration options when requesting facets through the [facets]{IQuery.facets} array.
+ */
+export interface IFacetOptions {
+  /**
+   * Indicates whether the facet values should be returned in their current order.
```

Comment:
The suggestion I was referring to:

Mm... does this option affect the order of facet values or of the facets themselves? If it's the former, I'd use the same description as in the Search API Swagger:

> Whether facets should be returned in the same order they were requested.
>
> Note: Setting this to true prevents automatically reordering facets using Coveo ML and index scores.